nixos/modules/services/mail/dovecot.nix

```diff
@@ -712,7 +699,6 @@ in
       wantedBy = [ "multi-user.target" ];
       restartTriggers = [
         cfg.configFile
-        modulesDir
```

Comment:
So no more automatic restart/reload when the modules change? :/

Reply:
Oh right, *that* is the actual functional reason for that having been there instead of being just vibes. I tried to avert it 🤷

Now that you say it, I think this part should be explicitly reverted.